src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelectorAll('li');
+const container = document.querySelector('ul');
+
+container.innerHTML = '';
+
+const emplList = getEmployees(sortList(list));
```

Comment:
The `sortList` function is called correctly, but ensure that the sorted list is appended back to the DOM. Currently, the sorted list is not being used to update the DOM.